lib/Graphics/graphics_funcs.cpp

```diff
@@ -201,6 +201,12 @@ void GraphicsHelps::getMaskFromRGBA(FIBITMAP *&image, FIBITMAP *&mask)
                                FreeImage_GetGreenMask(image),
                                FreeImage_GetBlueMask(image));
 
+    if(!mask)
+    {
+        pLogCritical("OOM when extracting mask!");
```

Comment:
Please write fully "Out of memory", just make such messages more obvious for newbies who attempt to read the log files.

Reply:
Done, thank you!